### PR TITLE
[github-actions] add Develop workflow

### PR DESCRIPTION
**Pull request by simonlingoogle**
This PR adds the `Develop` workflow to verify development related scripts work.

### CODE REVIEW

File: .github/workflows/develop.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        if: "github.ref != 'refs/heads/master'"
+
+  develop:
+    name: Develop (Go${{ matrix.go }},Py${{matrix.python-versoin}},${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        go: [1.11, 1.14]
+        python-versoin: [3.6, 3.8]
```
Comment (ctan-g) [inline .github/workflows/develop.yml:46]: Typo in `python-versoin`.
```suggestion
        python-version: [3.6, 3.8]
```

File: .github/workflows/develop.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  cancel-previous-runs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: rokroskar/workflow-run-cleanup-action@master
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        if: "github.ref != 'refs/heads/master'"
+
+  develop:
+    name: Develop (Go${{ matrix.go }},Py${{matrix.python-versoin}},${{ matrix.os }})
```
Comment (ctan-g) [inline .github/workflows/develop.yml:41]: Should match `build` actions
```suggestion
    name: Develop (Go ${{ matrix.go }}, Py ${{matrix.python-versoin}}, ${{ matrix.os }})
```
(Should also add a space in `test` script after `Py` too).

File: script/setup-dev [hunk 3]
```diff
@@ -118,13 +118,20 @@ install_go_bindata()
...
+    python3 -m pip install grpcio-tools==1.31.0 --user
+}
+
 install_protoc
 install_protoc_gen_go
 install_protoc_gen_grpc_web
 install_nodejs
 install_go_bindata
+install_py_libraries
+install_pretty_tools
```
Comment (ctan-g) [inline script/setup-dev:136]: Where is this function?

Comment (simonlingoogle) [reply script/setup-dev:136]: > Where is this function?

it's defined in utils.sh

File: .github/workflows/develop.yml (hunk 1 above)
Comment (simonlingoogle) [reply .github/workflows/develop.yml]: fixed

File: .github/workflows/develop.yml (hunk 2 above)
Comment (simonlingoogle) [reply .github/workflows/develop.yml]: I intentionally make the `name` more concise so that we can see more of it in the PR checks UI.

**Pull request merged**